src/slack/conversation.go

```diff
@@ -60,3 +62,15 @@ func (c *conversation) Get(key string) ([]string, bool) {
 	value, ok := c.data[key]
 	return value, ok
 }
+
+// ClearConversation delete current conversation history
+func (c *conversation) ClearConversation(key string, convo *conversation) bool {
+	// Key exists in the map delete it from map and return true
+	if _, ok := convo.data[key]; ok {
```

Comment:
See line 26 and how it accesses the key. It uses the `c.Get` method. Again, this is a blocking operation so that only one method will access the map at a given time.